gtp2ogs.js

```diff
@@ -672,9 +675,14 @@ class Bot {
         //
         this.firstmove = false;
 
-        this.command("genmove " + this.game.my_color,
-            (move) => {
-                move = typeof(move) == "string" ? move.toLowerCase() : null;
+	this.command(cmd, (line) => {
+	    line = typeof(line) == "string" ? line.toLowerCase() : null;
```

Comment:
I know it was this way before, but should the second option for the tenary operator be `""` rather than `null`? Otherwise the `split` below is going to throw an exception.

Reply:
I'd do this in another PR / commit then. One change at a time ...

Reply:
Oh I see your point, the old code was already using null. So leave that behavior as-is and we can PR it separately than the rest of these changes, correct?